static/js/context/auth.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, FC, ReactNode, useContext } from "react";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+
+const checkAuth = async () =>
+  await fetch("/1.0/certificates").then((response) => response.status !== 403);
```

Comment:
What is the point of waiting the response here? You could just return the promise as it is :)

Reply:
Ah, sorry, I didn't understand you were talking about the `async` / `await` thing. You're right. That can be removed and I did it with the last commit. Actually, it was a "leftover" because initially I wasn't using `useQuery`, I was fetching directly instead.